src/index.ts

```diff
@@ -199,12 +199,24 @@ class TestcafeTestrailReporter {
           return formatted;
         })
         .join("\n");
-
-      this.results.push({
-        case_id: caseId,
-        status_id: testStatus.value,
-        comment: `Test ${testStatus.text}\n${errorLog}`,
-      });
+      let testDuration;
+      if (testRunInfo.durationMs > 0) {
+        testDuration = (testRunInfo.durationMs / 1000).toString() + "s";
+      }
+      if (testDuration) {
+        this.results.push({
+          case_id: caseId,
+          status_id: testStatus.value,
+          comment: `Test ${testStatus.text}\n${errorLog}`,
+          elapsed: testDuration,
+        });
+      } else {
+        this.results.push({
+          case_id: caseId,
+          status_id: testStatus.value,
+          comment: `Test ${testStatus.text}\n${errorLog}`,
+        });
+      }
```

Comment:
Oh, could you please change `toStrictEqual` -> `toEqual` in those tests? This will solve the issue as `toEqual` should ignore undefined values when comparing the objects.

Reply:
Ok I do that